inf-clojure.el

```diff
@@ -199,6 +199,29 @@ has been found.  See also variable `inf-clojure-buffer'."
       (unless no-error
         (error "No Clojure subprocess; see variable `inf-clojure-buffer'"))))
 
+(defun inf-clojure-set-repl (always-ask)
+  "Set an inf clojure buffer as the active repl.
+If in a repl already, use that unless a prefix is used (or
+ALWAYS-ASK).  Otherwise get a list of all active inf-clojure
+repls and offer a choice.  Recommended to rename buffers as they
+are created with `rename-buffer`."
+  (interactive "P")
+  (cl-flet ((inf-clojure-repl-p () (and (derived-mode-p 'inf-clojure-mode)
```

Comment:
I'd suggest having this and a `inf-clojure-repls` fucntion to be top-level functions. I can imagine they'd be useful in some other situations as well.